### PR TITLE
Fixed panel titles and colours to match wireframes

### PR DESCRIPTION
**Pull request by lili2311**

### CODE REVIEW

File: frontend/src/sass/pages/_home.scss [hunk 1]
```diff
@@ -25,7 +25,7 @@
     color: $colr-txt-accent;
 }
 
-.pnl-causes, .pnl-marketers, .pnl-marketing, .pnl-signup, .pnl-contact-us, .pnl-talkabout, .pnl-hero  {
+.pnl-marketing, .pnl-marketers, .pnl-signup, .pnl-contact-us, .pnl-talkabout, .pnl-hero, .pnl-causes  {
```
Comment (Icicleta) [inline frontend/src/sass/pages/_home.scss]: is the order important.. here you just changed order right?

Comment (lili2311) [inline frontend/src/sass/pages/_home.scss]: the order of comma separated selectors is not important no. It will apply to all of them regardless of order.

Comment (Icicleta) [inline frontend/src/sass/pages/_home.scss]: so why would you change it?, just curious :)

**Comment by Icicleta** [comment]
👍

**Pull request merged**